features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1780,6 +1782,15 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }, [ initialValues ]
     );
 
+    useEffect(
+        () => {
+            setCurrentBindingType(initialValues?.accessToken?.bindingType
+                ? initialValues.accessToken.bindingType
+                : isFAPIApplication ? SupportedAccessTokenBindingTypes.CERTIFICATE
+                    : metadata?.accessTokenBindingType?.defaultValue
+                ?? SupportedAccessTokenBindingTypes.NONE);
```

Comment:
Let's implement this logic using if-else rather than nested ternary conditions. Improve readability.